packages/app/src/device-manager/observers/usb-device-attach.observer.ts

```diff
@@ -3,28 +3,37 @@
  * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
  */
 
-import usb from "usb"
 import { Observer } from "App/core/types"
-import { PortInfoValidator } from "App/device-manager/validators"
 import { DeviceManager } from "App/device-manager/services"
+import { PortInfoValidator } from "App/device-manager/validators"
+const outboxTime = 3000
```

Comment:
Why `outbox`? 🤔  Maybe a `time` is a good variable name?
```suggestion
const time = 3000
```

Reply:
Maybe something like `intervalTime`?

Reply:
renamed 9e2d742a8b4d419f433d38b40c08828f004c23bf